projects/elife.yaml

```diff
@@ -959,3 +950,34 @@ iiif:
     vagrant:
         ports:
             1261: 80
+
+redirects:
+    description: redirect various domains to the canonical elifesciences.org
+    subdomain: redirects
+    intdomain: null
+    formula-repo: https://github.com/elifesciences/redirects-formula
+    aws:
+        type: t2.nano
+        ports:
+            - 22
+            - 80
+            - 443
+    aws-alt:
+        prod:
+            cloudfront:
+                certificate_id: ASCAIEHY4AGS453E4IUJK
```

Comment:
same as https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L96 I understood this is just an ID, and without the right AWS credentials/account is not accessible